apps/meteor/server/settings/accounts.ts

```diff
@@ -711,7 +711,7 @@ export const createAccountSettings = () =>
 				i18nLabel: 'Notification_volume',
 			});
 
-			await this.add('Accounts_Default_User_Preferences_voipRingerVolume', 100, {
+			await this.add('Accounts_Default_User_Preferences_callRingerVolume', 100, {
```

Comment:
I'd ask to not change the setting `_id`, this usually has not the effect you'd expect, this doesn't actually _rename_ the setting, it actually creates a new one.. so workspaces which already have the old one will actually have two..

I understand the `_id` meaning may look wrong, but we should deal with it.. the correct way to "rename" a setting is via migration, which we cannot do during regular release cycles..

Reply:
can't we create this new one for now and remove the old setting in the next major?

Reply:
in order to do that we had to create a migration to update the new value with previous one, right?